reference/widgets.v3.yml

```diff
@@ -1231,11 +1231,19 @@ definitions:
         type: string
         description: name for each element in the list
       thumbnail:
-        type: string
+        type: object
         description: used to display an image stored at the specified attribute name
+        properties:
+          type:
+            type: string
+            example: image
+          valueKey:
+            type: string
+            example: imageUrl.src
       schema:
         type: array
         description: The schema used for each element in the array.
+        items: {}
```

Comment:
I think this is `widgetSchemaTab|widgetSchemaArray`

Reply:
@bc-jcha This is `widgetSchemaArray`. The `items` keyword is required in arrays per [Swagger documentation](https://swagger.io/docs/specification/data-models/data-types/#array). The value of `items` is a schema that describes the type and format of array items.

Reply:
Per our offline discussion, api-specs repo already has the items empty, so we will proceed with the current code for now.

Reply:
Merging, but we should specify the type for items.